linkedin-clone/src/components/PageDetails.js

```diff
@@ -150,15 +287,21 @@ const EditPostButton = ({ onClick }) => {
 };
 
 const DeletePostButton = ({ onClick }) => {
-  const handleClick = () => {
+  const handleClick = async () => {
     if (window.confirm("Are you sure you want to delete this post? This action cannot be undone.")) {
```

Comment:
Nice use of confirmation prompt. Can you add them for delete group and leave group?

Reply:
sure, approve this PR and I'll create a new branch/PR for it